packages/framework/src/Resources/views/Admin/Content/Customer/edit.html.twig

```diff
@@ -1,8 +1,8 @@
 {% extends '@ShopsysFramework/Admin/Content/Customer/detail.html.twig' %}
 
-{% block title %}- {{ 'Editing customer'|trans }} - {{ user.fullName }}{% endblock %}
+{% block title %}- {{ 'Editing customer'|trans }} - {{ customerUser.fullName }}{% endblock %}
```

Comment:
```suggestion
{% block title %}- {{ 'Editing customer user'|trans }} - {{ customerUser.fullName }}{% endblock %}
```
also translation dump will be needed

Reply:
As I wrote above. I am not sure if we really want to change translations.

Reply:
Ok, I get it